copter/source/docs/aerobatic-multicopter.rst

```diff
@@ -0,0 +1,42 @@
+.. _aerobatic-multicopter:
+
+============================
+Aerobatic/Racing Multicopter
+============================
+
+**this page is a work in progress and is incomplete currently**
+
+With the proper vehicle configuration and careful tuning, a precise, aggressive multicopter can be obtained for racing or aerobatics in ACRO mode.
```

Comment:
not a blocker of course but we could make "Acro" a link back to the flight mode page https://ardupilot.org/copter/docs/acro-mode.html#acro-mode